csv_export.php

```diff
@@ -68,16 +68,8 @@
 
 $t_filename = csv_get_default_filename();
 
-# Send headers to browser to activate mime loading
-
-# Make sure that IE can download the attachments under https.
-header( 'Pragma: public' );
-
-header( 'Content-Type: text/csv; name=' . urlencode( file_clean_name( $t_filename ) ) );
-header( 'Content-Transfer-Encoding: BASE64;' );
-
-# Added Quotes (") around file name.
-header( 'Content-Disposition: attachment; filename="' . urlencode( file_clean_name( $t_filename ) ) . '"' );
+csv_emit_headers( $t_filename );
```

Comment:
I am quite sure there are apps that are not able to deal with a BOM in CVS.
There is no clear specification for CSV https://tools.ietf.org/html/rfc4180
A check box in UI to add BOM could be a solution for it.